### DISCUSSION TITLE
Swipe motion config?

### DISCUSSION THREAD

**Discussion by sijow** [YaLTeR/niri#301] [Q&A] [answered]
Is there a way to configure the swipe motions? I'd like to have non-natural scrolling with the touchpad but natural for swipes. I thought I'd just put a minus sign in the swipe gestures but can't find a way to do that.

**Comment by YaLTeR** [comment, marked as answer]
There's no configuration for swipes at the moment

**Reply by sijow** [reply]
Thanks for the quick answer, I'll try to get use to natural scrolling :) And thanks for working on niri! It feels impressively solid already.

**Reply by YaLTeR** [reply]
Hope you enjoy it :)
